### DISCUSSION TITLE
How to animate other parts of the hierarchy using the new Layout Animations API?

### DISCUSSION THREAD

**Discussion by brunohkbx** [software-mansion/react-native-reanimated#3279] [Q&A]
I've been trying to replace some animations made with RN `LayoutAnimation` with the new reanimated API.

In my understanding, with Reanimated I can animate views either `entering` or `exiting` phase and how the layout behaves whereas RN `LayoutAnimations.configureNext` expects the following attrs: `duration`, `create`, `update`, `delete`.  It's not clear to me what's the equivalent of the attr `update`.

Here's what I tried - using the [Participant List example](https://github.com/software-mansion/react-native-reanimated/blob/main/Example/src/LayoutReanimation/AnimatedList.tsx).
I'd like to make the ScrollView smooth move down when I delete one of the participants and the content size changes - In the video below it feels like the TextInputs "jumps" to adjust to the new size - How to add a springify effect?

https://user-images.githubusercontent.com/6487206/171313386-b5fb12a7-1291-4b11-bfee-096ebd3db4ed.mp4

Now notice how smooth the ScrollView scrolls when I delete one item, It's using `LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut)`

https://user-images.githubusercontent.com/6487206/171314746-b31cc965-ce37-4936-bf8c-f61be8ec4e0d.mp4

Is it possible to achieve the same animation? I tried using `Animated.ScrollView` with `layout={Layout.springify()}` but it didn't work.

Snack example
https://snack.expo.dev/@bruno.castro/reanimated-layout-animation

Maybe related #2734?

**Comment by kesha-antonov** [comment]
Hi
Did you find a solution?
